docs/sources/mimir/configure/configure-shuffle-sharding/index.md

```diff
@@ -88,7 +89,56 @@ The Grafana Mimir shuffle sharding implementation provides the following benefit
 - **Zone-awareness**<br />
   When you enable [zone-aware replication](../configure-zone-aware-replication/), the subset of instances selected for each tenant contains a balanced number of instances for each availability zone.
 
-### Ingesters shuffle sharding
+### Partitions shuffle sharding
+
+Without shuffle sharding when using ingest storage, the Grafana Mimir distributor divides the received series among all active partitions. With shuffle-sharding the distributor divides each tenant's series among multiple partitions.
+
+Configuring shuffle sharding on the write path automatically also enables shuffle sharding on the read path too.
+
+You can configure the default shard size for all tenants by setting - `-ingest-storage.ingestion-partition-tenant-shard-size=<size>` or its YAML equavalent. The shard size can be overridden on a per-tenant basis by setting `ingestion_partitions_tenant_shard_size` in the `overrides` section of the runtime configuration.
+
+#### Partitions shuffle sharding on the write path
+
+With partition shuffle sharding, the distributor divides each tenant's series among `-ingest-storage.ingestion-partition-tenant-shard-size` number of partitions.
+
+#### Partitions shuffle sharding read path
+
+When shuffle sharding is enabled for the write path, the read path automatically uses the same configuration to query only the relevant partition owners (ingesters). The querier uses the ShuffleShardWithLookback algorithm to:
+
+- Include all partitions in the tenant's current shard
+- Include partitions that were recently part of the tenant's shard (within the lookback period)
+- Include INACTIVE partitions that are transitioning to a read-only state
+- Exclude PENDING partitions that have not yet received any traffic.
+
+This ensures query consistency during partition lifecycle transitions such as adding or removing ingesters. For more information about the hash ring and the state of partitions see [Hash ring](../../references/architecture/hash-ring/)
+
+#### Rollout strategy
+
+If you’re running a Grafana Mimir cluster with shuffle sharding disabled, and you want to enable it for the ingesters, use the following rollout strategy to avoid missing querying for any series currently in the ingesters:
```

Comment:
```suggestion
If you’re running a Grafana Mimir cluster with shuffle sharding disabled, and you want to enable it for the ingesters, use the following rollout strategy to avoid missing queries for any series currently in the ingesters:
```